### ISSUE TITLE
Multiple folders or symlinks

### ISSUE THREAD

**Issue by AlphaX2** [blissd/fotema#210]
Hi there,

I want to suggest that Fotema allows you to add multiple folders or at least follows symlinks, in this case I could create a folder filled with symlinks to the folders I want to add - it's a bit more like workaround, but better than having just one folder.

Thank you very much!

**Comment by NelvinBHV** [comment]
Hi AlphaX2, if you like to you symlinks in fotema, you can download the patch here:

https://github.com/NelvinBHV/PortagePirats/tree/main/media-gfx/fotema/files

**Comment by Kartoffelbauer** [comment]
yes, such a feature would be really great, especially in times of cloud service providers like ownCloud or Nextcloud, where you have multiple media folders and you want to combine their contents in one gallery

**Comment by mbnoimi** [comment]
+1
